### PR TITLE
[BACKLOG-35528] convert log4j.xml to log4j2.xml with file rename

### PR DESCRIPTION
**Pull request by ddiroma**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 3m 22s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

File: core/src/test/resources/log4j2.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="ERROR">
+   <Appenders>
+      <Console name="CONSOLE" target="SYSTEM_ERR">
```
Comment (bennychow) [inline core/src/test/resources/log4j2.xml:4]: Missing Threshold filter set to ERROR

File: pentaho/src/main/resources/log4j2.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="DEBUG">
+   <Appenders>
+      <Console name="CONSOLE">
```
Comment (bennychow) [inline pentaho/src/main/resources/log4j2.xml:4]: Missing Threshold filter set to DEBUG

File: proxy/src/main/resources/log4j2.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="DEBUG">
+   <Appenders>
+      <Console name="CONSOLE">
```
Comment (bennychow) [inline proxy/src/main/resources/log4j2.xml:4]: Missing Threshold filter set to DEBUG

**Review by bennychow — CHANGES_REQUESTED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 3m 39s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Review by bennychow — APPROVED** [review]

**Pull request merged**